control_plane/attachment_service/src/compute_hook.rs

```diff
@@ -33,6 +36,7 @@ struct ComputeHookNotifyRequestShard {
 #[derive(Serialize, Deserialize, Debug)]
 struct ComputeHookNotifyRequest {
     tenant_id: TenantId,
+    stripe_size: Option<ShardStripeSize>,
```

Comment:
@mattpodraza The assumption here is that if we include this new field before the corresponding CP change, it will simply be ignored.

Reply:
That's correct, we ignore unknown fields